packages/analytics/shippers/elastic_v3/server/src/server_shipper.test.ts

```diff
@@ -122,7 +122,7 @@ describe('ElasticV3ServerShipper', () => {
       shipper.reportEvents(events);
       shipper.optIn(true);
       const counter = firstValueFrom(shipper.telemetryCounter$);
-      setLastBatchSent(Date.now() - 10 * MINUTES);
+      setLastBatchSent(Date.now() - 10 * SECONDS);
```

Comment:
If I understand the [revised README](https://github.com/elastic/kibana/pull/143323/files#diff-9af9194571431124a3ca92057431c182768556b721a7abcab43c9561d19c4699R25) correctly, we should be sending every 10s now, yes? So this line is setting us up for "we last sent 10 seconds" ago. 

But the next line still advances time by 10 minutes, which should no longer be needed, right? 

And the comment on line 120 still mentions the 10 minute window, which is not strictly true any longer, yes?